.scalafmt-common.conf

```diff
@@ -0,0 +1,51 @@
+# This file was automatically generated by sbt-lucuma using the
```

Comment:
I thought this file was going to be in the Lucuma sbt plugin?

Reply:
It is in the sbt-lucuma plugin. As the comment says, it should not be changed because it's automatically generated (copied out of sbt-lucuma plugin).

The reason it has to be copied into every repo is because otherwise IDEs won't know where to read the scalafmt config from. More information:
- https://github.com/typelevel/sbt-typelevel/issues/103#issue-1114631735
- https://scalameta.org/scalafmt/docs/installation.html#share-configuration-between-builds

Actually, I guess we can put the file into `.gitignore` and re-generate it every sbt is reloaded.